src/Adapter/Ftp.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Bolt\Filesystem\Adapter;
+
+use League\Flysystem\Adapter\Ftp as FtpAdapter;
+use League\Flysystem\Config;
+
+/**
+ * FTP adapter.
+ *
+ * @author Gawain Lynch <gawain.lynch@gmail.com>
+ */
+class Ftp extends FtpAdapter
+{
+    /** @var int */
+    protected $directoryPerm = 0750;
```

Comment:
Yes, I had copy/pasta what I was using (over & over & over again) without thinking. It is a small but important distinction and you are 100% correct! Done.